README_FUZZER.md

```diff
@@ -0,0 +1,8 @@
+How to look into fuzzer issues
+==============================
+
+* Look into the output of the test case at: https://oss-fuzz.com/testcases?showall=1 (Nathan, I am not sure how to look into a specific testcase coming from the logs in CI; could you chime in and put the procedure here ?).  Use the blosc.oss.fuzz@gmail.com account so as to access it.
```

Comment:
@nmoinvaz Could you please complete the testcase lookup procedure here?  Thanks in advance!

Reply:
Oss-fuzz.com is only for people on the CC list in google/oss-fuzz project configuration yaml. Otherwise people can only view the `CI Fuzz` CI run output which is what I did in the case of this PR.

![image](https://user-images.githubusercontent.com/1364432/99994678-2e37e680-2d6e-11eb-96ac-4f26ab2a1cc7.png)
![image](https://user-images.githubusercontent.com/1364432/99994735-427be380-2d6e-11eb-9553-c3afa362cba1.png)

Reply:
I see now.  I updated the `README_FUZZER.md` accordingly.  Thanks!